### PR TITLE
Specify the output file name for wget command

### PR DESCRIPTION
**Pull request by benquike**
Without specifying the output filename, wget may saves the downloaded file to
a file with an unexpected name (e.g., when the file is hosted on AWS), which
leads to build failure.

This patch enforces wget to save downloaded files to files with desired names.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -284,9 +284,7 @@ ifeq ($(LLVM_BUILD),$(S2E_BUILD))
 
 stamps/clang-binary: $(CLANG_BINARY) | stamps
 	tar -xmf $<
-	mkdir -p $(S2E_PREFIX)
-	cp -r $(CLANG_BINARY_DIR)/* $(S2E_PREFIX)
-	rm -r $(CLANG_BINARY_DIR)/*
+	cp -r $(CLANG_BINARY_DIR) $(S2E_PREFIX)
```
Comment (vitalych) [inline Makefile:287]: Not clear why this change is needed. It looks unrelated to the commit, so either remove it or put it in a separate commit.

**Review by vitalych — CHANGES_REQUESTED** [review]
Modify commit message title to:
`makefile: specify the output file name for wget command`

File: Makefile (hunk 1 above)
Comment (benquike) [reply Makefile:287]: This is not needed.

**Comment by benquike** [comment]
updated

**Review by vitalych — APPROVED** [review]
